.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "spellright.language": [
+        "en_GB"
```

Comment:
Why GB instead of US? I believe we have always preferred US in code, and have over the years also moved our documentation from GB to US.